crates/transaction-generator-lib/src/args.rs

```diff
@@ -15,9 +15,9 @@ pub enum TransactionTypeArg {
     CoinTransferWithInvalid,
     AccountGeneration,
     AccountGenerationLargePool,
-    NftMintAndTransfer,
     PublishPackage,
-    CreateNewAccountResource,
+    AccountResourceSmall,
```

Comment:
Can we rename to AccountResource32B, AccountResource1K, etc.

Also can we also have 2k, 4k, 5k?